app/src/Http/Response/ScriptResponse.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Discussion\Http\Response;
+
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\HttpFoundation\Response;
+
+class ScriptResponse extends Response
```

Comment:
I couldn't find an easy way in HttpFoundation to simply run a particular PHP script. Included files can use $this, and can return their own Responses for chaining.